electron/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "author": "Wire Swiss <wireapp@wire.com>",
   "dependencies": {
+    "@wireapp/certificate-check": "0.0.2",
```

Comment:
Needs to become 0.0.3 (to serve the definition file), no?

Reply:
Yes, I was just waiting for wireapp/wire-desktop-packages#50 to be merged.